### PR TITLE
Turn Submit proposal tab into a modal

### PR DESCRIPTION
**Pull request by fr4ctalbit**
1. Convert Submit proposal tab into a modal component
2. Add a button to open up this modal

### CODE REVIEW

**Comment by jacogr** [comment]
We are duplicating the account to submit from.

Current tab-screens -

propose-1

The new modal (we duplicate the from) -

propose-2

File: packages/app-democracy/src/Overview/Propose.tsx [hunk 1]
```diff
@@ -3,40 +3,68 @@
...
-import { RouteComponentProps } from 'react-router';
-import { withRouter } from 'react-router-dom';
 import { createType } from '@polkadot/types';
-import { Button, Extrinsic, InputAddress, InputBalance, TxButton, TxComponent } from '@polkadot/react-components';
+import { Button, Extrinsic, InputAddress, InputBalance } from '@polkadot/react-components';
 import { withApi, withMulti } from '@polkadot/react-api';
 
-import translate from './translate';
+import translate from '../translate';
+import TxModal, { TxModalState, TxModalProps } from '@polkadot/react-components/TxModal';
```
Comment (jacogr) [inline packages/app-democracy/src/Overview/Propose.tsx]: (nit) move this to just below the `'@polkadot/react-components'` imports on line 11

Comment (fr4ctalbit) [reply packages/app-democracy/src/Overview/Propose.tsx]: Yep, sure. Didn't see the duplication of account cause didn't create one before testing :)
I've removed that input and adjusted imports

**Comment by fr4ctalbit** [comment]
Fixed it. It was the order of arguments in txParams.

**Comment by jacogr** [comment]
Thank you, will take a final look in the next 60-odd minutes.

**Pull request merged**